package.json

```diff
@@ -40,7 +40,7 @@
   },
   "devDependencies": {
     "bower": "1.5.2",
-    "enb": ">= 0.15.0 < 1.0.0",
+    "enb": ">= 1.2.0 < 2.0.0",
```

Comment:
For write file.

I could also just copypaste it here, if we need to support enb 0.15.x.